course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineFragment.kt

```diff
@@ -168,14 +168,24 @@ class CourseOutlineFragment : Fragment() {
                         viewModel.resumeSectionBlock?.let { subSection ->
```

Comment:
We can also move this and the other block (in DatesFragment) to their respective ViewModel, but that's up to you now. 😄